### PR TITLE
Cut from representation and transfer learning sections

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
### Purpose

Related to #87 - cutting from the representation learning and transfer learning sections. I, admittedly, did not check for the actual % of words that I reduced here. (Have folks been doing that?)

### Directions for reviewers

* Do the cuts/edits retain meaning? If so...
* Are there points/messages that you would consider extraneous at this time?

### Manuscript checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#figures)
- [x] All terms that need to be added to glossary have been added to #50 
- [x] There are no new spelling errors identified by the Manubot spellchecker

### CODE REVIEW

File: content/03.prior-knowledge.representation-and-transfer.md [hunk 1]
```diff
@@ -1,55 +1,52 @@
...
-
-Whether or not a learned representation or set of features is _useful_ depends on the task at hand.
-In a supervised setting, it may be sufficient for a feature to distinguish between classes, but if we hope to use a feature for biological discovery, we may prioritize intepretability.
-For example, learned features in the medical imaging domain may be a series of edges that constitute a blood vessel formation that discriminates between disease states or the learned features may not align with characteristics recognized as important by domain experts at all.
-From transcriptomics data, learned features could be coordinated sets of genes involved in a biological process that are descriptive in some way [@doi:10.1038/s41467-020-14666-6], but do not necessarily distinguish cases from controls in our study.
+Supervised neural networks used in medical imaging studies [@doi:10.1016/j.procs.2016.07.014] (reviewed in [@doi:10.1016/j.zemedi.2018.11.002] and [@doi:10.1098/rsif.2017.0387]), which are trained to predict labels or classes, are also an example of representation learning.
```
Comment (allaway) [inline content/03.prior-knowledge.representation-and-transfer.md]:
```suggestion
Supervised neural networks used in medical imaging studies [@doi:10.1016/j.procs.2016.07.014] (reviewed in [@doi:10.1098/rsif.2017.0387]), which are trained to predict labels or classes, are also an example of representation learning. 
```
I wonder if one of these citations could go? I left the earlier one behind.

File: content/03.prior-knowledge.representation-and-transfer.md [hunk 2]
```diff
@@ -1,55 +1,52 @@
...
-In a supervised setting, it may be sufficient for a feature to distinguish between classes, but if we hope to use a feature for biological discovery, we may prioritize intepretability.
-For example, learned features in the medical imaging domain may be a series of edges that constitute a blood vessel formation that discriminates between disease states or the learned features may not align with characteristics recognized as important by domain experts at all.
-From transcriptomics data, learned features could be coordinated sets of genes involved in a biological process that are descriptive in some way [@doi:10.1038/s41467-020-14666-6], but do not necessarily distinguish cases from controls in our study.
+Supervised neural networks used in medical imaging studies [@doi:10.1016/j.procs.2016.07.014] (reviewed in [@doi:10.1016/j.zemedi.2018.11.002] and [@doi:10.1098/rsif.2017.0387]), which are trained to predict labels or classes, are also an example of representation learning. 
+Learned features in the medical imaging domain may be a series of edges that constitute a blood vessel formation that discriminates between disease states.
```
Comment (allaway) [inline content/03.prior-knowledge.representation-and-transfer.md]:
```suggestion
Learned features in the medical imaging domain may be a series of edges representing a blood vessel formation that discriminates between disease states.
```

File: content/03.prior-knowledge.representation-and-transfer.md [hunk 3]
```diff
@@ -1,55 +1,52 @@
...
+Features learned from transcriptomic data could be coordinated sets of genes involved in a biological process that are descriptive in some way [@doi:10.1038/s41467-020-14666-6].
 
 In the rare disease domain, Dincer et al. leveraged publicly available acute myeloid leukemia (AML) gene expression data to improve the prediction of _in vitro_ drug responses [@doi:10.1101/278739]. 
 The authors trained a variational autoencoder (VAE) on AML data that had been collected over time without the phenotypic information they were interested in–in this case, drug response.
```
Comment (allaway) [inline content/03.prior-knowledge.representation-and-transfer.md]:
```suggestion
The authors trained a variational autoencoder (VAE) on AML data that had been collected over time without the phenotypic information they were interested in (drug response). 
```

File: content/03.prior-knowledge.representation-and-transfer.md [hunk 4]
```diff
@@ -1,55 +1,52 @@
...
-
-Taken together, the DeepProfile [@doi:10.1101/278739] and MultiPLIER [@doi:10.1016/j.cels.2019.04.003] results suggest transfer learning can be beneficial for studying rare diseases. 
+(Here "unseen" refers to the fact that these diseases were not in the training set.)
+PLIER is a matrix factorization approach that takes prior knowledge in the form of gene sets or pathways and gene expression data as input; some latent variables learned by the model will align with input gene sets  [@doi:10.1038/s41592-019-0456-1].
+We demonstrated that training on larger collections of randomly selected samples produced models that captured a larger proportion of input gene sets and better distinguished closely related signals, which suggests larger training sets produced models that are more suitable for biological discovery [@doi:10.1016/j.cels.2019.04.003].
```
Comment (allaway) [inline content/03.prior-knowledge.representation-and-transfer.md]:
```suggestion
We demonstrated that training on larger collections of randomly selected samples produced models that captured a larger proportion of input gene sets and better distinguished closely related signals, which suggests that larger training sets produced models that are more suitable for biological discovery [@doi:10.1016/j.cels.2019.04.003].
```

File: content/03.prior-knowledge.representation-and-transfer.md [hunk 5]
```diff
@@ -1,55 +1,52 @@
...
+
+Though models trained on generic compendia had appealing properties, that alone does not guarantee suitability for describing rare diseases.
+We must examine the relevance of learned features to the disease under study.
+In Taroni et al., we found that the expression of latent variables that could be matched between the MultiPLIER model and a dataset-specific model were well-correlated, particularly when latent variables were associated with input gene sets [@doi:10.1016/j.cels.2019.04.003].
+Despite the absence of AAV from the training set, MultiPLIER was able to learn a latent variable where the genes with the highest contributions encode antigens that the antineutrophil cytoplasmic antibodies (ANCA) form against in AAV and with higher expression in more severe disease [@doi:10.1002/art.27398].
```
Comment (allaway) [inline content/03.prior-knowledge.representation-and-transfer.md:45]: I wonder if this sentence could be omitted? Might be overly detailed for this paper, not sure.
```suggestion
```

**Review by allaway — APPROVED** [review]
Looking good! I left a few suggested changes.

File: content/03.prior-knowledge.representation-and-transfer.md (hunk 5 above)
Comment (jaclyn-taroni) [reply content/03.prior-knowledge.representation-and-transfer.md:45]: Without this sentence, I think I would have to remove 

> –the same ANCA antigen genes are components of neutrophilic granule development that is likely captured or assayed in the collection of transcriptomic data used for training

In the next sentence. But the sentiment

> The utility of this approach stems from the fact that biological processes are often _shared_ between conditions

may be enough, thoughts?

Comment (jaclyn-taroni) [reply content/03.prior-knowledge.representation-and-transfer.md:45]: I'm going to plan leave this for now and see what #88 brings ✂️

Comment (allaway) [reply content/03.prior-knowledge.representation-and-transfer.md:45]: Sorry, just saw your follow up comment. I will consider this when I work on #88!

**Pull request merged**